fhf-datafangst-client/src/components/BenchmarkData/BenchmarkCards.tsx

```diff
@@ -0,0 +1,231 @@
+    if (type === BenchmarkType.totalTime) {
+      benchmarkModal.title = "Total tid";
+      benchmarkModal.description =
+        "Total tid er regnet som tiden mellom havneavgang og havneanløp.";
+      metric = totalTimeMean > 24 ? "Dager" : "Timer";
+      data =
+        totalTimeMean > 24 ? totalTimes.map((value) => value / 24) : totalTimes;
+    } else if (type === BenchmarkType.fishingHours) {
+      benchmarkModal.title = "Fiske tid";
+      benchmarkModal.description =
+        "Fiske tid er regnet som summen av tiden brukt under hver fangstmelding.";
+      metric = totalTimeMean > 24 ? "Dager" : "Timer";
+      data =
+        fishingHoursMean > 24
+          ? fishingHours.map((value) => value / 24)
+          : fishingHours;
+    } else if (type === BenchmarkType.fishingDistance) {
+      benchmarkModal.title = "Fiske distanse";
+      benchmarkModal.description =
+        "Fiske distanse er regnet ut basert på vms/ais meldingene som ble sendt under hver fangstmelding";
+      metric = fishingDistanceMean > 1852 ? "Nautisk Mil" : "Meter";
+      data =
+        fishingDistanceMean > 1852
+          ? fishingDistance.map((value) => value / 1852)
+          : fishingDistance;
+    } else if (type === BenchmarkType.fishingWeight) {
+      benchmarkModal.title = "Total vekt";
+      benchmarkModal.description = "Total vekt er basert på total landet vekt";
+      metric = fishingWeightMean > 1000 ? "Tonn" : "Kilo";
+      data =
+        fishingWeightMean > 1000
+          ? fishingWeight.map((value) => value / 1000)
+          : fishingWeight;
+    } else {
+      return;
+    }
+    dispatch(
+      setBenchmarkHistoric({ metric, xAxis: getTripDates(trips), data }),
+    );
+    dispatch(setBenchmarkModal(benchmarkModal));
+  };
+
+  return (
+    <Grid
+      container
+      spacing={3}
+      sx={{ marginTop: "3vh", backgroundColor: "primary.main" }}
+    >
+      <Grid item xs={6}>
+        <Box>
+          <BenchmarkCard
+            title="Total tid"
+            avatar={<AccessTimeIcon />}
+            value={totalTimeMean > 24 ? totalTimes[0] / 24 : totalTimes[0]}
+            description="Siste tur"
+            primary_color={
+              totalTimes[0] > totalTimeMean ? "#6CE16A" : "#93032E"
+            }
+            secondary_value={
+              totalTimeMean > 24 ? totalTimeMean / 24 : totalTimeMean
+            }
+            secondary_description={
+              "Gjennomsnitt siste " + numHistoric.toString() + " turer"
+            }
+            metric={totalTimeMean > 24 ? "Dager" : "Timer"}
+            tooltip="Regnet ut basert på dine por og dep meldinger."
+            onClick={() => handleClick(BenchmarkType.totalTime)}
+          />
+        </Box>
+      </Grid>
+      <Grid item xs={6}>
+        <Box>
+          <BenchmarkCard
+            title="Fiske tid"
+            avatar={<PhishingRoundedIcon />}
+            value={
+              fishingHoursMean > 24 ? fishingHours[0] / 24 : fishingHours[0]
```

Comment:
You can do a bit better than this.
Using decimals for days, like 38.40, isn't really common (how many hours is 0.40 days?).
Maybe do something similar to https://github.com/orcalabs/fhf-datafangst/blob/78726231cc285a3cb02b0b7782fa2e44d265e8be/fhf-datafangst-client/src/components/CurrentTripMenu/CurrentTripMenu.tsx#L116